ScanCommand/src/main/resources/cli-help/ballerina-scan.help

```diff
@@ -0,0 +1,69 @@
+NAME
+        ballerina-scan - Static code analyzer
+
+SYNOPSIS
+        bal scan [OPTIONS] [<package>|<source-file>]
+
+DESCRIPTION
+        Compiles and performs static code analysis, prints results to the console,  and reports results.
+
+        Analyze the source code defined in each module when compiling a package or analyze the given source file
+        when compiling a single BAL file.
+
+        Note: Analyzing individual BAL files of a package is not allowed.
+
+
+OPTIONS
+        --target-dir=<path>
+                Target directory path for saving analysis reports. (only for ballerina build projects)
+
+        --scan-report
+        	    Generates an HTML report containing the analysis results. (only for ballerina build projects)
```

Comment:
```suggestion
        Generates an HTML report containing the analysis results. (only for ballerina build projects)
```

Reply:
The tab spaces for the descriptions look inconsistent.

Reply:
Fixed in [da46b97](https://github.com/ballerina-platform/static-code-analysis-tool/pull/3/commits/da46b977ecd4bc8ac563fb490e97eed889b71f54)